packages/styling/src/interfaces/ISerifs.ts

```diff
@@ -0,0 +1,21 @@
+import { IRawStyle } from '@uifabric/merge-styles';
+
+/**
+ * @internal
+ * Experimental interface for decorative styling in a theme.
+ */
+export interface ISerifs {
+  depth4: IRawStyle;
+  depth8: IRawStyle;
+  depth16: IRawStyle;
```

Comment:
There doesn't seem to be a depth32 step, not sure if that's intentional or part of Fluent design.

Reply:
That's intentional. :)

Reply:
I'd really rather not use 4, 8, 16, etc. Either:

we snap to 100 scale; (depth100, depth200, etc)

or, we do small/medium/large/etc

Reply:
Note that 4, 8, 16 etc are the names used in Fluent across the company today.

Reply:
ISerifs?????? Ok that is a damaging name. Someone in the future is going to waste a lot of time trying to figure that out in the code. Let's pick a different name please?

Reply:
How about ISurfaces, or ISurfaceStyles?